recipe/build.sh

```diff
@@ -15,8 +19,22 @@ do
 	    --enable-symlinks \
 	    --enable-termcap \
 	    --with-termlib \
-	    $USE_WIDEC \
+	    $WIDEC_OPT \
 	    --with-terminfo-dirs=/usr/share/terminfo
     make
     make install
+
+    # Provide headers in `$PREFIX/include` and
+    # symlink them in `$PREFIX/include/ncurses`
+    # and in `$PREFIX/include/ncursesw`.
+    HEADERS_DIR="${PREFIX}/include/ncurses"
+    if [ "${USE_WIDEC}" = true ];
+    then
+        HEADERS_DIR="${PREFIX}/include/ncursesw"
+    fi
+    for HEADER in $(ls $HEADERS_DIR);
+    do
+        mv "${HEADERS_DIR}/${HEADER}" "${PREFIX}/include/${HEADER}"
+        ln -s "${PREFIX}/include/${HEADER}" "${HEADERS_DIR}/${HEADER}"
+    done
```

Comment:
This is how we handle moving and symlinking. We don't have to worry about clobbering files as the headers are the exact same in all cases.